front/src/common/Map/Settings/MapSettingsSpeedLimits.tsx

```diff
@@ -39,31 +61,30 @@ function FormatSwitch(props) {
     );
   };
 
-  const getTagsList = async (_zoom, _params) => {
-    try {
-      const tagsList = await get(`/editoast/infra/${infraID}/speed_limit_tags/`);
+  const changeTagsListToObject = (tagsListArray: string[]) => {
+    const tagsListObject = [{ key: t('noSpeedLimitByTag'), value: t('noSpeedLimitByTag') }].concat(
+      tagsListArray.slice().map((tag) => ({ key: tag, value: tag }))
```

Comment:
No need for slice() here because map() already returns a new array (it does not mutate the existing array)
```suggestion
      tagsListArray.map((tag) => ({ key: tag, value: tag }))
```

Reply:
Done